resources/views/tasks/show.blade.php

```diff
@@ -1,14 +1,144 @@
+                                    <img class="mr-2 avatar_placeholder" src="{{ asset('img/avatar-placeholder.gif') }}"/>
+                                </div>
+                                <div class="form-group col">
+                                <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-10"></div>
+                                <div class="form-group col text-right">
+                                <button class="btn btn-success">comment</button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div> 
+        
+        <div class="col-3">
+            <div class="card card-body border-0">
+                <div class="row">
+                    <h4 class="text-primary col-9">Task Details</h4>
+                    <i class="fas fa-caret-square-up text-secondary col"></i>
+                </div>
+                <div class="text-secondary">Assigned to:</div>
+                <div class="">Name Name</div>
+                <span class="text-secondary mt-2">Assigned:</span>
+                <div class="mt-1">01/01/1001 11:11</div>
+
+                <div class="text-secondary mt-5">Date Created:</div>
+                <div class="mt-1">3 days ago</div>
+                <div class="text-secondary mt-2">Due Date:</div>
+                <div class="mt-1">10 mins ago</div>
+                <div class="text-secondary mt-2">Last Modified:</div>
+                <div class="mt-1 mb-4">24hrs</div>
+            </div>
+            <br>
+            <div class="card card-body border-0">
+                <div class="row">
+                    <h4 class="text-primary col-9">Request Details</h4>
+                    <i class="fas fa-caret-square-up text-secondary col"></i>
+                </div>
+                <div class="text-secondary">Process:</div><div class="mt-1">Name Name</div>
+                <div class="text-secondary mt-2">Created by:</div><div class="mt-1">01/01/1001 11:11</div>
+                <br>
+                <div class="text-secondary ">Date Created:</div><div class="mt-1">3 days ago</div>
+                <div class="text-secondary mt-2">Created:</div><div class="mt-1">10 mins ago</div>
+                <div class="text-secondary mt-2">Last Modified:</div><div class="mt-1">24hrs</div>
+                <div class="text-secondary mt-2">Duration:</div><div class="mt-1 mb-4">24hrs</div>
+            </div>
+        </div>
+    </div>
+    <br>
+    <br>
 </div>
 @endsection
 
 @section('js')
 <script src="{{mix('js/tasks/show.js')}}"></script>
 @endsection
+
+@section('css')
+<style lang="scss" scoped>
+
+.avatar_placeholder {
```

Comment:
There should be a global style for the avatars. Use that instead.

Reply:
complete